public/assets/locales/en/translation.json

```diff
@@ -1,4 +1,5 @@
 {
   "welcome_to_react": "Welcome to React and react-i18next",
-  "variable_example": "Hi, my name is {{name}}"
+  "variable_example": "Hi, my name is {{name}}",
+  "footer_copyright": "AquaSharp - Scientific Circle PIMI 2023"
```

Comment:
zamiast footer_copyright napiszmy dokładnie to co ma być wyświetlane tylko po angielsku, w pl/translation.json też napiszmy po angielsku